packages/cozy-sharing/src/components/ShareByLink.jsx

```diff
@@ -229,9 +245,9 @@ class ShareByLink extends React.Component {
                         this.deleteShareLink()
                       }}
                     >
-                      <Text className="u-pomegranate">
+                      <Typography className="u-pomegranate" variant="body1">
```

Comment:
```suggestion
                      <Typography className="u-error" variant="body1">
```

Reply:
There is another instance of pomegranate in the same file, if it could be also taken care of via a className='u-error', same in Recipient.jsx.